src/line_with_indices_index.rs

```diff
@@ -0,0 +1,73 @@
+#[derive(Debug, Clone)]
+pub struct LineWithIndicesArray<T: AsRef<str>> {
+  pub line: T,
+  pub prefix_array: Box<[u32]>,
```

Comment:
Can you put a comment on what this stores?

Reply:
Done